prometheus-exporter/cloudwatch/README.md

```diff
@@ -0,0 +1,15 @@
+
+
+# Docker version for Apache exporter for Prometheus
```

Comment:
This readme should be for Cloudwatch, right? we can delete it if it's not ready yet